static/docs/commands-reference/repro.md

```diff
@@ -19,25 +19,26 @@ positional arguments:
 
 ## Description
 
-If the [DVC-file](/doc/user-guide/dvc-file-format) (`target`) is omitted,
-`Dvcfile` will be assumed.
+`dvc repro` provides an interface to run the commands in a computational graph
+(a.k.a. pipeline) again, as defined in the stage files (DVC-files) found in the
+current workspace. (A pipeline is typically defined using the `dvc run` command,
+while data input nodes are defined by the `dvc add` command.)
 
-`dvc repro` provides an interface to run the commands in the computational graph
-(a.k.a. pipeline) again, as defined in the corresponding stage files (DVC-files)
-found in the current workspace. By default, this command recursively searches
-the pipeline stages, starting from the `targets`, to determine which ones have
-changed. Then it runs again the corresponding commands. The pipeline is mostly
-defined using the `dvc run` command, while data input nodes are defined by the
-`dvc add` command.
+There's a few ways to restrict the stages that will be run again by this
+command: by specifying stage file(s) as `targets`, or by using the
+`--single-item`, `--pipeline`, or `--cwd` options.
```

Comment:
... and other options, no need to name them all, but we now have a few more -R, --downstream

Reply:
I did already notice and add `--downstream` to the list in a recent comment... But let me review again...

Reply:
Ok, list shortened!